app/models/peep.rb

```diff
@@ -0,0 +1,17 @@
+class Peep
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :content, Text
+  property :author, Text
+  property :created_at, DateTime
+
+  def display_date_and_time
+    text = self.created_at.to_s
```

Comment:
The fact that the method name here has "display" in it is indicative of a code smell - I would argue that the model's job is not to handle how data is displayed, just to return it from the database. Instead this could be a view helper